### PR TITLE
Docstring pandas.series.diff

### PR DESCRIPTION
**Pull request by adatasetaday**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
################################## Validation ##################################
################################################################################

Docstring for "pandas.Series.diff" correct. :)
```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
@adatasetaday were you assigned both DataFrame.diff and Series.diff? OK to do as one PR, but wanted to make sure we don't conflict with someone else.

**Comment by TomAugspurger** [comment]
Ah I see https://github.com/pandas-dev/pandas/pull/20227

could you remove the dataframe changes from this branch so we don't ahve merge conflicts?

**Comment by TomAugspurger** [comment]
LGTM once you remove the dataframe changes.

**Comment by adatasetaday** [comment]
@TomAugspurger Yes, I was assigned diff for both Series and DataFrame. I made sense since it were very similar.

I fixed the commit.

File: pandas/core/series.py [hunk 1]
```diff
@@ -1607,16 +1607,61 @@ def cov(self, other, min_periods=None):
...
+            values.
 
         Returns
         -------
         diffed : Series
+
+        See Also
+        --------
+        Series.pct_change: Percent change over given number of periods.
+        DataFrame.diff: First discrete difference of object
```
Comment (jreback) [inline pandas/core/series.py]: add Series.shift

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by adatasetaday — COMMENTED** [review]
Series.shift added

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @adatasetaday very nice!

**Pull request merged**